aio/.eslintrc.json

```
@@ -0,0 +1,100 @@
+{
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "tsconfig.json",
+          "e2e/tsconfig.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "extends": [
+        "plugin:@angular-eslint/ng-cli-compat",
+        "plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "aio",
+            "style": "kebab-case"
+          }
+        ],
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "aio",
+            "style": "camelCase"
+          }
+        ],
+        "dot-notation": "error",
+        "indent": "off",
+        "no-tabs": "error",
+        "no-restricted-syntax": [
+          "error",
+          {
+              "selector": "CallExpression[callee.object.name='console'][callee.property.name=/^(debug|info|time|timeEnd|trace)$/]",
+              "message": "Unexpected property on console object was called"
+          },
+          {
+            "selector": "CallExpression[callee.name='fdescribe']",
+            "message": "Don't keep jasmine focus methods."
+          },
+          {
+            "selector": "CallExpression[callee.name='fit']",
+            "message": "Don't keep jasmine focus methods."
+          }
```

I know it might have come from the automatic conversion, but you can condense these two into:

```suggestion
          {
            "selector": "CallExpression[callee.name=/^(fdescribe|fit)$/]",
            "message": "Don't keep jasmine focus methods."
          }
```

---

good point! :slightly_smiling_face: 

This actually was not automated but it myself :sweat_smile:  (as the tslint ban rule doesn't have a 1-to-1 alternative), I did it separately as I thought that the two rules had each a different/specific message, I did notice that the error message was the same but forgot to combine the rules! thanks for noticing it! :slightly_smiling_face:

---

😮 Maybe you could have follow the steps described [here](https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint) 😄

---

yeah I did!

sorry if my comment was unclear :stuck_out_tongue_winking_eye: , I did follow that, what I meant is that the schematics failed to the tslint ban rules one rule (which did what I am doing here), they just provided such line in the log file: 
```
* tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "ban".
```

so I wrote this rule myself :slightly_smiling_face: 
(the schematics did an awesome job but after those I did need to do quite some manual adjustments and stuff)